Packs/PhishUp/Integrations/PhishUp/PhishUp.yml

```
@@ -0,0 +1,95 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: 6437c006-1d86-4cdc-89d5-23305f207e3a
+  version: -1
+configuration:
+- display: API KEY
+  name: apikey
+  required: true
+  type: 0
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- additionalinfo: If there is any Phishing activity in mail, what should PhishUp do?
+  defaultvalue: Nothing
+  display: PhishUp Playbook Actions
+  hidden: false
+  name: phishup-playbook-action
+  options:
+  - Nothing
+  - ' Delete Mail'
+  - ' Move to SPAM'
+  required: true
+  type: 15
+description: PhishUp prevents phishing attacks, protects your staff and your brand
+  with AI
+display: PhishUp
+name: 6437c006-1d86-4cdc-89d5-23305f207e3a
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Single URL for phishup investigation
+      isArray: false
+      name: Url
+      required: true
+      secret: false
+    deprecated: false
+    description: Single url investigation
+    execution: false
+    name: phishup-investigate-url
+    outputs:
+    - contextPath: PhishUp.Result
+      description: response types "Clean", "Phish", "Error"
+      type: String
+    - contextPath: PhishUp.Score
+      description: Phishup Engine Url Score
+      type: Number
+  - arguments:
+    - default: false
+      description: Bulk Url Arry
+      isArray: true
+      name: Urls
+      required: false
+      secret: false
+    deprecated: false
+    description: Bulk url investigation
+    execution: false
+    name: phishup-investigate-bulk-url
+    outputs:
+    - contextPath: PhishUp.AverageResult
+      description: If any harmfull URL in list this value will return "Phish" otherwhise
+        will return "Clean", if any error happens will return "Error"
+      type: String
+    - contextPath: PhishUp.Results
+      description: Stringified Urls Result List
+      type: String
+  - deprecated: false
+    description: Get chosen action from PhishUp instance
+    execution: false
+    name: phishup-get-chosen-action
+    outputs:
+    - contextPath: PhishUp.Action
+      description: Chosen action from PhishUp instance
+      type: String
+  dockerimage: demisto/python3:3.9.9.25564
+  feed: false
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+tests:
+- No tests (auto formatted)
+fromversion: 5.5.0
```

Please change fromversion to be 6.0.0
```suggestion
fromversion: 6.0.0
```